Sources/PT.PM.Common/ILanguageHandler.cs

```diff
@@ -0,0 +1,7 @@
+namespace PT.PM.Common
+{
+    public interface ILanguageHandler : ILoggable
```

Comment:
What is the meaning of this interface? I see it used only in `AntlrBaseHandler`. So, there is no big need.